docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # The base image (buildbox:latest) is built by running `make -C build.assets`
 # from the base repo directory $GOPATH/gravitational.com/teleport
-FROM quay.io/gravitational/teleport-buildbox:go1.17.2
+FROM quay.io/gravitational/teleport-buildbox:teleport9
```

Comment:
@tcsc I changed it to use the new buildbox tag here, is this ok?

Reply:
Not sure why it wasn't that already. Reasonably sure I put a PR in for it when I changed the rest.